hooks/storage/badger/badger.go

```diff
@@ -136,7 +137,7 @@ func (h *Hook) OnWillSent(cl *mqtt.Client, pk packets.Packet) {
 // updateClient writes the client data to the store.
 func (h *Hook) updateClient(cl *mqtt.Client) {
 	if h.db == nil {
-		h.Log.Error().Err(storage.ErrDBFileNotOpen)
+		h.Log.Error("", "error", storage.ErrDBFileNotOpen.Error())
```

Comment:
I think these `.Error()` calls can removed as @thedevop noted.

Reply:
Good catch.

Reply:
Pushed changes to fix all references.